js/interface.js

```diff
@@ -3224,6 +3224,16 @@ $('.browse-files').on('click', function (e) {
   });
 });
 
+// Scroll accordion tab to the top
+$(document).ready(function (){
```

Comment:
why is `$(document).ready` required in this case?

Reply:
It is not necessary but I used it to make sure that the document is fully loaded.
If it is not need, I will change it without $(document).ready.

Reply:
fixed